docs/guides/quick-start/miranum-connect/process-start-quickstart.md

```diff
@@ -0,0 +1,95 @@
+---
+id: process-start-quickstart
+title: Start a Process Quickstart
+sidebar_label: Start a Process
+description: "This section introduces the concept of starting a process using the Miranum Connect API."
+---
+
+# Starting a Process with Miranum Connect
+
+In this quickstart guide, we will cover how to use Miranum Connect to initiate a process. This feature allows you to start a process from your code, thereby offering great flexibility and control over your workflows.
+
+:::info What you will learn
+- How to add the necessary dependencies to use Miranum Process.
+- How to start a process instance from your code.
+:::
+
+:::note
+This guide assumes familiarity with Java and Maven.
+:::
+
+## Add the Necessary Dependencies
+
+To start, we need to include the `process-api` dependency in our project. This is the core API that enables you to start a process with Miranum Connect. Include the following snippet in your `pom.xml` file:
+
+```xml
+<dependency>
+ <groupId>io.miragon.miranum</groupId>
+ <artifactId>process-api</artifactId>
+ <version>0.2.0</version>
```

Comment:
💭 We have a specific version mentioned here but a`${miranum.worker.version}` below. I think we should unify this. (e.g. with `${miranum.version} `)